pkg/aws/cloudfront/types.go

```diff
@@ -24,10 +24,13 @@ type Client struct {
 type CreateInvalidationOutput struct {
 	InvalidationId string
 	Status         string
+	CreateTime     time.Time
+	Paths          []string
 }
 
 type GetInvalidationOutput struct {
-	CreateTime time.Time
-	Status     string
-	Paths      []string
+	InvalidationId string
+	Status         string
+	CreateTime     time.Time
+	Paths          []string
```

Comment:
I had to add a few parameters to satisfy the requirements of [InvalidationResponse](https://github.com/kanopy-platform/cdnvalidator/blob/main/internal/core/v1beta1/types.go#L22)

Reply:
nit: InvalidationID